builds/jazz-build-module/utility-loader.groovy

```diff
@@ -201,4 +201,19 @@ def getAzureAccountInfo(service_config){
 	return dataObj;
 }
 
+def getRoleDetails(lambdaARN, region, credsId) {
```

Comment:
Look at my other comments, we might not need it..

Reply:
Replied